recipe-server/.eslintrc

```diff
@@ -22,6 +22,7 @@ rules:
   no-underscore-dangle: [off]
   no-use-before-define: [warn, { functions: false }]
   prefer-const: [warn]
+  new-cap: [off]
```

Comment:
This is for Immutable, right? I think I'd prefer to use `new Map()` and `new List()` instead, and keep this rule. In my testing it works the same (though maybe I've missed something?)